frontend/ui/widget/checkbutton.lua

```diff
@@ -130,14 +130,14 @@ end
 function CheckButton:check()
     self:initCheckButton(true)
     UIManager:setDirty(self.parent, function()
-        return "partial", self.dimen
+        return "fast", self.dimen
```

Comment:
Those two (partial -> fast) I'm less convinced about.
Not quire sure where they actually end up being used...